Makefile

```diff
@@ -111,8 +111,7 @@ dev-install:
 
 .PHONY: uninstall
 uninstall:
-	-$(PIP_UNINSTALL) -y nni
-	-$(PIP_UNINSTALL) -y nnictl
+	-cd build && $(PIP_UNINSTALL) -y nni
```

Comment:
please also remove the `build` directory.

Reply:
Have removed